beacon_node/network/src/sync/network_context.rs

```diff
@@ -418,11 +418,11 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
         };
         let request_id = RequestId::Sync(sync_id);
 
-        trace!(
+        debug!(
```

Comment:
Wondering why we want this as a debug. We log from multiple places when we need to make a roots request. In the ideal case, this should always result in a roots request.

Reply:
> We log from multiple places when we need to make a roots request. In the ideal case, this should always result in a roots request.

this is part of what was confusing me in the deneb logs. we were logging during construction of the lookup but the introduction of the `delayed_lookup_service` for example meant we may not actually end up making the request, and if we do we may be requesting a smaller set of blobs than we originally thought because more blobs arrived over gossip during the delay. So removing the other logs and elevating this one makes it more clear when we make requests and what exactly we are requesting for. Also this log lets us see the blob indices we're requesting each time we retry which IMO is useful.

Reply:
Fair enough